cli/src/main/kotlin/OrtServerMain.kt

```diff
@@ -23,33 +23,35 @@ import com.github.ajalt.clikt.command.SuspendingNoOpCliktCommand
 import com.github.ajalt.clikt.command.main
 import com.github.ajalt.clikt.core.Context
 import com.github.ajalt.clikt.core.subcommands
-import com.github.ajalt.clikt.parameters.groups.OptionGroup
-import com.github.ajalt.clikt.parameters.groups.provideDelegate
-import com.github.ajalt.clikt.parameters.options.convert
-import com.github.ajalt.clikt.parameters.options.option
-import com.github.ajalt.clikt.parameters.options.required
 import com.github.ajalt.clikt.parameters.options.versionOption
 
 import kotlin.system.exitProcess
 
 import kotlinx.serialization.json.Json
 
 import org.eclipse.apoapsis.ortserver.client.OrtServerClient
-import org.eclipse.apoapsis.ortserver.client.OrtServerClientConfig
+import org.eclipse.apoapsis.ortserver.client.auth.AuthenticationException
 import org.eclipse.apoapsis.ortserver.model.ORT_SERVER_VERSION
 
 const val COMMAND_NAME = "osc"
 
 suspend fun main(args: Array<String>) {
-    OrtServerMain().main(args)
+    runCatching {
+        OrtServerMain().main(args)
+    }.getOrElse { t ->
+        if (t.cause is AuthenticationException) {
+            println("Authentication failed. Please run '${COMMAND_NAME} auth login' to authenticate.")
```

Comment:
Maybe [this](https://ajalt.github.io/clikt/exceptions/#handling-exceptions-manually) would be the right approach to be able to use `echo` instead of `print` here.